tests/EventsClientFunctionalTest.php

```diff
@@ -18,7 +19,11 @@ private function getClient(): EventsClient
     {
         return new EventsClient(
             (string) getenv('TEST_NOTIFICATION_API_URL'),
-            (string) getenv('TEST_MANAGE_API_APPLICATION_TOKEN')
+            (string) getenv('TEST_MANAGE_API_APPLICATION_TOKEN'),
+            [
+                'backoffMaxTries' => 3,
```

Comment:
Vidim ve vsech testech stejny `'backoffMaxTries' => 3`. Vede me to k myslence - budeme chtit nastavovat v ruznych projektech ruzne hodnoty? Nebylo by dobre tu mit nejaky rozumny default a prenastaveni udelat optional?

Reply:
no jelikož se tenhle klient použivá jen v démonovi, tak odpověd je asi jasná :) Ale jo, jinak platí, že můžeme chtít nastavovat různé hodnoty a právě protože je tam ten default, tak na to věčně zapomínáme

Reply:
jo, zapominame, ale default je ted "no retry", ne? kdyby se nechal default treba jako 3, neni to reasonable default, ktery funguje 90% casu a jen ve vyjimkach chces nastavit neco jinyho?